### PR TITLE
modernize gradle and xtend

### PR DESCRIPTION
**Pull request by cdietrich**

### CODE REVIEW

File: examples/states-xtext/language-server/gradle/maven-deployment.gradle [hunk 1]
```diff
@@ -1,11 +1,12 @@
-//see https://docs.gradle.org/current/userguide/maven_plugin.html
-apply plugin: 'maven'
+publishing {
```
Comment (cdietrich) [inline examples/states-xtext/language-server/gradle/maven-deployment.gradle:1]: is this used at all?

File: examples/states-xtext/language-server/build.gradle [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
-		classpath 'org.xtext:xtext-gradle-plugin:2.0.8'
+		classpath 'org.xtext:xtext-gradle-plugin:4.0.0'
 	}
 }
 
 subprojects {
-	ext.xtextVersion = '2.26.0'
+	ext.xtextVersion = '2.34.0'
 	ext.sprottyVersion = '0.9.1'
 	ext.elkVersion = '0.8.1'
```
Comment (cdietrich) [inline examples/states-xtext/language-server/build.gradle:14]: newest in 0.9.1

File: examples/states-xtext/language-server/build.gradle [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
 	dependencies {
-		classpath 'org.xtext:xtext-gradle-plugin:2.0.8'
+		classpath 'org.xtext:xtext-gradle-plugin:4.0.0'
 	}
 }
 
 subprojects {
-	ext.xtextVersion = '2.26.0'
+	ext.xtextVersion = '2.34.0'
 	ext.sprottyVersion = '0.9.1'
```
Comment (cdietrich) [inline examples/states-xtext/language-server/build.gradle:13]: newest is 1.1.1

**Review by spoenemann — APPROVED** [review]
Thanks!

**Pull request merged**